fastapi-challenges/jwt-auth/devashmit-day17/auth.py

```
@@ -0,0 +1,13 @@
+from datetime import datetime,timedelta,timezone
+from jose import JWTError,jwt
+from passlib.context import CryptContext
+SECRET="change-me-long-random-secret";ALGO="HS256";EXP=30
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to: "SECRET"

(S105)